src/ParkDetails/ParkDetails.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ParkDetails.css";
+
+function ParkDetails({ foundPark }) {
+  if (!foundPark) {
+    return null;
+  }
+
+  return (
+    <div className="park-details-container">
+      <div className="park-details-title">
+        <h2 className="single-title">{foundPark.fullName}</h2>
+      </div>
+      <section className="image-container">
+        <img
+         className="single-image"
+         src={foundPark.images[1].url}
+         alt={foundPark.images[0].altText}
+        />
+      </section>
+      <section className="park-info">
+        <div className="park-details-activities">
+          <h3 className="single-activities">Activities</h3>
+          {/* <p>{foundPark.activities}</p> */}
+        </div>
+        <div className="park-details-weather">
+          <h3 className="single-weather">Weather</h3>
```

Comment:
did the foundPark.activities not work?

Reply:
It was just an empty array. I figured we can just hold off for now until we get the BE hooked up.